tests/e2e/esm/index.test.js

```diff
@@ -544,14 +544,27 @@ describe('e2e esm tests', async () => {
   });
 
   describe('ANT', async () => {
-    const processId = 'YcxE5IbqZYK72H64ELoysxiJ-0wb36deYPv55wgl8xo';
+    // ANT v8 source
+    const processId = 'oQ4GNTed8cnNw-H5olq606gCFd5MbGSV4NTmfpIW4FI';
     const ant = ANT.init({
       process: new AOProcess({
         processId,
         ao: aoClient,
       }),
     });
 
+    it('should be able to get info on old ant', async () => {
```

Comment:
we changed the getRecords API - we want to test that the SDK correctly maps an old ANTs record response...nothing has changed about the info handler - so this isn't validating anything

Reply:
[77601b2](https://github.com/ar-io/ar-io-sdk/pull/259/commits/77601b2fc9117a2a4b9ff1226fa3f329e8a74e1d)